<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowserWindow</class>
 <widget class="QDockWidget" name="FileBrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Browser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../res/FileBrowser.qrc">
     <normaloff>:/oxygen/places/user-home.png</normaloff>:/oxygen/places/user-home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="statusTip">
    <string>Go to the home folder</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../../../misc/Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/go-previous.png</normaloff>:/oxygen/actions/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="statusTip">
    <string>Go to the previous folder or file</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../../misc/Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/go-next.png</normaloff>:/oxygen/actions/go-next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="statusTip">
    <string>Go to the next folder or file</string>
   </property>
  </action>
  <action name="actionParent">
   <property name="icon">
    <iconset resource="../res/FileBrowser.qrc">
     <normaloff>:/oxygen/actions/go-up.png</normaloff>:/oxygen/actions/go-up.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parent</string>
   </property>
   <property name="statusTip">
    <string>Go to the parent folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../misc/Core/res/Core.qrc"/>
  <include location="../res/FileBrowser.qrc"/>
 </resources>
 <connections/>
</ui>
